Formula/icalpal.rb

```diff
@@ -0,0 +1,31 @@
+class Icalpal < Formula
+  desc "Command-line tool to query a macOS Calendar for accounts, calendars, and events"
+  homepage "https://github.com/ajrosen/icalPal"
+  url "https://github.com/ajrosen/icalPal/archive/refs/tags/1.0.2.tar.gz"
+  sha256 "435cdf4098da06fa401e7abf40c1be28d0758f3f399bc2547e59bb0d80263bf5"
+  license "GPL-3.0-or-later"
+
+  uses_from_macos "ruby"
+
+  resource "testdata" do
+    url "https://github.com/ajrosen/icalPal/raw/main/test/testdata.tar.gz"
+    sha256 "298f95ce9d54c7434602bd6f3bd22aba67cfcc8cd14ddefe1549cf71a7c6edd4"
+  end
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "#{name}.gemspec"
```

Comment:
The formula is named `icalpal` but your gemspec file is named [`icalPal.gemspec`](https://github.com/ajrosen/icalPal/blob/1.0.2/icalPal.gemspec). So, the build on Linux failed with this:

```
==> gem build icalpal.gemspec
ERROR:  Couldn't find a gemspec file matching 'icalpal.gemspec' in /tmp/icalpal-20230225-8032-18yiq0h/icalPal-1.0.2
```

(This didn't fail on macOS because the filesystem is case-insensitive by default. However, a fix is required as case insensitivity is not necessarily true. It is fine not to use `#{name}` here.)